Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:14.15-alpine as build
```

Comment:
Se for seguir uma abordagem multistage [artigo no dev.to](https://dev.to/chrsgrrtt/dockerising-a-next-js-project-1ck5) que pode ser uma boa. Coloquei o Dockerfile no fim do artigo como exemplo.

---

```Dockerfile
FROM node:14.15-alpine AS base

WORKDIR /base

COPY package*.json ./
RUN npm install

COPY . .


FROM base AS build

ENV NODE_ENV=production

WORKDIR /build

COPY --from=base /base ./
RUN npm run build


FROM node:14.15-alpine

ENV NODE_ENV=production

WORKDIR /app

COPY --from=build /build/package*.json ./
COPY --from=build /build/.next ./.next
COPY --from=build /build/public ./public
RUN npm install next

EXPOSE 3000

CMD ["npm", "run", "start"]
```

Reply:
vlw, não conheço js ou o next, ai me mantive no minimo.
Já vou implementar os passos do artigo. 
Mas de verdade eu queria ter gerado um bin ou uma app autocontida para correr tudo.

Acabei de atualizar o Dockerfile, para a abordagem multi-stage.  ^^